Create create_release.yml

Shtoyan opened the pull request:
Fixes #56 

- [x] Tests pass
- [ ] Appropriate changes to README are included in PR

Hi. This PR adds a simple workflow to automate github releases:
- you make a new release with a new tag.
- workflow creates binaries automatically and adds to release.

I haven't updated README yet, because I wanted to clarify few things beforehand:

- You're currently only providing a `deb` package. Is it strictly required or would a universal linux executable be acceptable?
- What targets do you wantto support? I added all available github runners, with latest os versions.

Shtoyan commented:
@pgrace-google @adetaylor 
- Added `x86_64-apple-darwin` target.
- And an example how it will look like: https://github.com/Shtoyan/ripunzip/releases/tag/0.0.3

pgrace-google commented:
I would prefer to keep the deb package! but otherwise adding mac/windows support and binaries lgtm

the FR author is the one who requested additional targets, but i dont see why we cant support all the ones that have been added.

Shtoyan commented:
@pgrace-google got it. 

I updated the README and added deb variants: https://github.com/Shtoyan/ripunzip/releases/tag/2.0.3
In your case you match the package version with github tags, so tags in binary names will be the same.

Is this good enough or should I modify the action further?
